src/utils/taskHelper.ts

```diff
@@ -1,20 +1,20 @@
 import { botCache, Collection } from "../../deps.ts";
 import { Task } from "./../types/tasks.ts";
 
-let runningTasksList: Array<number> = [];
-
 const registerTasks = () => {
   for (const task of botCache.tasks.values()) {
-    runningTasksList.push(setInterval(() => task.execute(), task.interval));
+    botCache.runningTasks.push(
+      setInterval(() => task.execute(), task.interval)
+    );
   }
 };
 
 const clearTasks = () => {
-  for (const task of runningTasksList) {
+  for (const task of botCache.runningTasks) {
     clearInterval(task);
   }
   botCache.tasks = new Collection<string, Task>();
-  runningTasksList = [];
+  botCache.runningTasks = Array<number>();
```

Comment:
```suggestion
  botCache.runningTasks = [];
```
We have already casted the type as `number[]`, so we do not have to do it again.